packages/api/server/http.mts

```diff
@@ -45,6 +47,37 @@ router.use(express.json());
 
 router.options('/file', cors());
 
+/**
+ * Note things added with data.append will be 2: in stream, LLM res is 0: so you can
+ * add any data you need to pass to the client as this goes to trigger UI events or
+ * other actions.
+ */
+router.get('/ai/anthropic/generate', cors(), async (_req: Request, res: Response) => {
```

Comment:
Why are we hardcoding anthropic here?

Reply:
I can swap it, but you mentioned sticking to one model since they handled prompts differently and that 3.5 sonnet was the best so I went with it, nothing stops swapping the name doesn't particularly matter